app/presenters/error_message/presenter.rb

```diff
@@ -36,7 +36,7 @@ def error_is_for_govuk_has_one_association?(attribute)
     end
 
     def govuk_has_one_associations
-      @govuk_has_one_associations ||= %w[fixed_fee. graduated_fee. transfer_fee.] # add associations to the array as they are migrated - eg %w[graduated_fee. fixed_fee.]
+      @govuk_has_one_associations ||= %w[fixed_fee. graduated_fee. misc_fees. transfer_fee.] # add associations to the array as they are migrated - eg %w[graduated_fee. fixed_fee.]
```

Comment:
Does `misc_fees` have a has one association names?

Reply:
I thought so, from this file: `spec/validators/claim/advocate_claim_sub_model_validator_spec.rb`

![Screenshot 2022-07-15 at 11 18 49](https://user-images.githubusercontent.com/90189839/179204311-9709e949-463e-4d19-8984-cc02f6c6a620.png)

Reply:
Done ✅